app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
   ],
   "scripts": {
-    "postdeploy": "python app/manage.py migrate"
+    "postdeploy": "sh scripts/postdeploy.sh"
+  },
+  "env": {
+    "DJANGO_SECRET_KEY": {
+      "required": true
```

Comment:
Instead of bringing over the staging secret, can we use the generator option? 

`{ "generator": "secret" }`